modules/admin_manual/pages/release_notes.adoc

```diff
@@ -30,14 +31,144 @@
+
+To prevent files being locked infinitely, there is a mechanism that automatically expires locks after a certain time. The expiration time of locks can be configured via the "Manual File Locking" section in the menu:Settings[Admin > Additional] settings or using _occ_ commands:
+
+- _Default timeout for the locks if not specified (in seconds)_: Maximum lifetime of a lock set **via the web interface** (or by not specifying a timeout value when calling the WebDAV Locks API) +
+`occ config:app:set core lock_timeout_default --value 1800`
+
+- _Maximum timeout for the locks (in seconds)_: Maximum lifetime of locks which is allowed to be set by calling the WebDAV Locks API +
+`occ config:app:set core lock_timeout_max --value 86400`
+
+By default locks set in the web interface will expire after 30 minutes. The expiration time is bound to the individual locks and can't be changed after locking. The maximum lock time by default is one day.
+
+TIP: Please change the lock expiration settings according to your needs. Usually you will only need to change the default timeout as that applies to locks set in the web interface.
+
+TIP: The user-facing components in the web interface are disabled by default. Administrators can enable the feature by executing the following _occ_ command: +
+`occ config:app:set files enable_lock_file_action --value yes`
+
+=== Changes to the ownCloud Marketplace
+
+The ownCloud Marketplace is opening up. With the release of Server 10.5, all apps available on the Marketplace (including ownCloud Enterprise apps) are now also available for download and installation via the Market app. This change facilitates the process of getting started with ownCloud Server and of evaluating Enterprise functionality. Additionally, it allows updates for all apps to be obtained from the ownCloud Marketplace and ensures running up-to-date versions.
+
+=== Changes in Delivery
+
+In line with the changes to the Marketplace the deliverables for ownCloud Server have been unified. Previously there were different Tarball bundles, Docker images and Linux packages for the Community (bare minimum) and Enterprise (all supported apps) Editions.
+Starting with Server 10.5 there are the following bundles which are shipped via tarball, Docker images and Linux packages:
+- minimal bundle for the Server and required components, semantically versioned (`ownCloud-10.5.0`)
+- complete bundle for the Server and all supported apps, including the Enterprise features, not semantically versioned as it always contains the latest versions of all supported apps (`ownCloud-complete-<date>`)
+
+The availability of Enterprise features now only depends on the license key.
+As part of this process, the former `enterprise_key` app has been deprecated and is not shipped anymore. New versions for all Enterprise apps have been released and included in the Server 10.5 complete bundle. These are needed to work with PHP 7.4 and the new licensing mechanism (see below).
```

Comment:
```suggestion
As part of this process, the former `enterprise_key` app has been deprecated and is not used anymore. New versions for all Enterprise apps have been released and included in the Server 10.5 complete bundle. These are needed to work with PHP 7.4 and the new licensing mechanism (see below).
```

After the latest iterations, we do still "ship" the `enterprise_key` app in the release bundle, but it does nothing useful.

Reply:
> ...has been deprecated and is not used anymore.

means removing from the build/tar file, or ? @pmaier1 pls decide/trigger

Reply:
We decided that we will deliver a "dummy/non-functional" `enterprise_key` app in the release tarball(s). This helps avoid possible upgrade problems if an admin had an old `enterprise_key` app and then thinks that they should manually enable it... They will have a "shiny new version" that, if it somehow gets enabled, will not actually do anything, and so avoid any possibility of harmful interaction with enterprise apps.

So technically `enterprise_key` app is shipped, but what is shipped as effectively a https://github.com/kelseyhightower/nocode app :)

Reply:
> ...the former `enterprise_key` app has been deprecated and is not used anymore.

I would be confused a bit when I read `deprecated and is not used anymore` and then find it in the shipped app list. Proposing to add a text that describes what you have written in nice words to make it clear why it is there (and may go away in future completely)

Reply:
https://github.com/owncloud/docs/pull/2697/commits/4322bb5ad20fd49dbf6f700e0a8e75d43f77dde4#diff-ba12788edde90b189cd9bb4d1b5492afR105

Reply:
Thanks for the correction @phil-davis. I overlooked that point.

Reply:
Agree, write some different words. The challenge is not to write so much that the reader gives up! Maybe:
```suggestion
As part of this process, the former `enterprise_key` app has been deprecated and a non-functional version is shipped. It will be removed completely in some future release. New versions for all Enterprise apps have been released and included in the Server 10.5 complete bundle. These are needed to work with PHP 7.4 and the new licensing mechanism (see below).
```